presto-main/src/test/java/io/prestosql/type/TestUuidOperators.java

```diff
@@ -33,6 +33,12 @@
 public class TestUuidOperators
         extends AbstractTestFunctions
 {
+    @Test
+    public void testRandomUuid()
+    {
+        tryEvaluateWithAll("uuid()", UUID);
```

Comment:
You could verify the value well-formedness.
eg it should be not null, and for test purposes we can assume it should not be all zeros.

Reply:
The non-null part is handled by the expected type assertion. I tried to add a test to validate they are unique, but the non-determinism makes it tricky due to the way the function asserts work.

Reply:
is there a ATQ's `computeActual` equivalent?